CorsixTH/Lua/hospital.lua

```diff
@@ -936,12 +982,32 @@ function Hospital:boilerBreakdown()
   end
 end
 
--- When the boiler has been repaired this function is called.
-function Hospital:boilerFixed()
-  self.radiator_heat = self.curr_setting
-  self.heating_broke = false
-  if self:isPlayerHospital() then
-    self.world.ui.adviser:say(_A.boiler_issue.resolved)
+--! Boiler broke down and work is done to get it fixed.
+function Hospital:_fixBoiler()
+  local heat_vars = self.heating
+
+  if not heat_vars.heating_broke then return end -- Not broken, done!
+
+  -- Repair the boiler or radiators, more handyman speeds up repair, see also github #490
```

Comment:
handymen

Reply:
I always have a bit of fun with this word :)